src/components/TypeBase/TypeBase.module.scss

```diff
@@ -397,3 +413,111 @@ h6,
     }
   }
 }
+
+/* ------------------------ Start Type2 ------------------------ */
+.Caption2 {
+  &.Theinhardt {
+    $font-size: 14;
+    $line-height: 16;
+    font-size: $font-size * 1px;
+    line-height: $line-height / $font-size;
+    letter-spacing: 0.0045em;
+  }
+}
+
+.Body2 {
+  &.Theinhardt {
+    $font-size: 17.5;
+    $line-height: 24;
+    font-size: $font-size * 1px;
+    line-height: $line-height / $font-size;
+    letter-spacing: 0;
+  }
+}
+
+.TitleSmall2 {
+  &.Theinhardt {
+    $font-size: 20;
+    $line-height: 26;
+    font-size: $font-size * 1px;
+    line-height: $line-height / $font-size;
+    letter-spacing: 0.001em;
+  }
+
+  &.Cambon {
+    $font-size: 24.5;
+    $line-height: 28;
+    font-size: $font-size * 1px;
+    line-height: $line-height / $font-size;
+    letter-spacing: -0.017em;
+  }
+}
+
+.TitleMedium2 {
+  &.Theinhardt {
+    @include for-phone-only {
+      /* Matches TitleSmall2 all viewports*/
+      $font-size: 20;
+      $line-height: 26;
+      font-size: $font-size * 1px;
+      line-height: $line-height / $font-size;
+      letter-spacing: 0.001em;
+    }
+
+    @include for-tablet-and-up {
+      $font-size: 24;
+      $line-height: 32;
+      font-size: $font-size * 1px;
+      line-height: $line-height / $font-size;
+      letter-spacing: -0.0125em;
+    }
+  }
+
+  &.Cambon {
+    $font-size: 32;
+    $line-height: 36;
+    font-size: $font-size * 1px;
```

Comment:
Nitpick:
I'm not a huge fan of this approach personally (I know it's what's in there now). I'd prefer to see sizes like 32 and 24 and what not shared more centrally and then pulled in here. Maybe even without the sass math.

Reply:
I added a note to address any improvements / refactors to the components to the next phase of this project in the blueprint in this PR: https://github.com/getethos/ethos-design-system/pull/318/files#diff-f15018e54270686bc2a5b593e49e6d33R84

I'll add this to the list.

Reply:
b8e22db